### PR TITLE
Update home.html

### PR DESCRIPTION
**Pull request by adambraimbridge**
**Problem** 
Not everyone knows what CTO stands for in this context, and even if they know it means "Chief Technology Officer", they wouldn't know the concept of a "CTO organisation" or what it is.

**Solution**
Rename "CTO organisation" to "within the remit of the Chief Technology Officer (CTO)"

**Do I like this solution?**
No.

### CODE REVIEW

**Comment by rowanbeentje** [comment]
Mmm, I raised this on https://github.com/Financial-Times/engineering-progression/issues/78 as well - fixing the acronym is great, but I’m not sure “under the remit of” helps much either! How can I find out whether I’m included in a remit?

**Comment by rowanmanning** [comment]
Does it make more sense to say "engineers in the Product and Technology department at the FT"? All of them are a bit wordy, but understand that it's not completely clear right now

**Comment by adambraimbridge** [comment]
**Update**
Trying "engineers in the FT's Product and Technology department" on for size.

**Do I like this solution?**
No.

**Comment by adambraimbridge** [comment]
I'm closing this PR because it's muddying the waters. See this issue for details: https://github.com/Financial-Times/engineering-progression/issues/78
